nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -218,12 +218,15 @@ These endpoints `/ipam/roles/`, `/dcim/rack-roles/` and `/dcim/device-roles/` ar
 |                       | `has_child_items`         | `has_children`                   | `/dcim/inventory-items/?has_children=True/False`                          |
 |                       | `tag`                     | `tags`                           | `/dcim/inventory-items/?tags=<slug>`                                      |
 | Location              | `tag`                     | `tags`                           | `/dcim/locations/?tags=<slug>`                                            |
+| ObjectPermission      | `group`                   | `groups`                         | `/users/permissions/?groups=<slug>`                                       |
+|                       | `group_id`                | `groups_id`                      | `/users/permissions/?groups_id=<id>`                                      |
```

Comment:
Did it end up not being possible to collapse these into a single `groups` NaturalKeyOrPKMultipleChoiceFilter?

Reply:
Yer, cause of `NaturalKeyOrPKMultipleChoiceFilter` not handling  Group pk integer field properly

Reply:
Thanks for the reminder. I don't think we have an issue open on this yet - would you mind opening one and adding appropriate TODO comments in the filterset code?